webapp/src/main/java/com/box/l10n/mojito/service/scheduledjob/ScheduledJobType.java

```diff
@@ -1,11 +1,13 @@
 package com.box.l10n.mojito.service.scheduledjob;
 
+import com.box.l10n.mojito.service.scheduledjob.jobs.ScheduledEvolveSync;
 import com.box.l10n.mojito.service.scheduledjob.jobs.ScheduledThirdPartySync;
 import com.box.l10n.mojito.service.scheduledjob.jobs.ScheduledThirdPartySyncProperties;
 
 public enum ScheduledJobType {
   THIRD_PARTY_SYNC(
-      ScheduledThirdPartySync.class.getName(), ScheduledThirdPartySyncProperties.class);
+      ScheduledThirdPartySync.class.getName(), ScheduledThirdPartySyncProperties.class),
+  EVOLVE_SYNC(ScheduledEvolveSync.class.getName(), ScheduledJobProperties.class);
```

Comment:
I think we should add `ScheduledEvolveSyncJobProperties` rather than making the class non abstract